modules/efk-logging-fluentd-undefined.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-fluentd.adoc
+
+[id="efk-logging-fluentd-undefined_{context}"]
+= Configuring how the log collector normalizes logs 
+
+Cluster Logging uses the link:https://github.com/ViaQ[ViaQ common data model] to deal with problem that arise from conflicts 
+and inconsistencies in log data collected from different subsystems.   
```

Comment:
Probably shouldn't mention that specific plugin for that specific collector.  Instead let's say something like this:
```
Cluster Logging uses a certain data model ([1],[2]), like a database schema, to store log records and their metadata in the logging store.  There are some restrictions on the data:

* there must be a `"message"` field containing the actual log message
* there must be a `"@timestamp"` field containing the log record timestamp in rfc3339 (preferably millisecond or better resolution) format
* there must be a `"level"` field with the log level e.g. `err`, `info`, `unknown`, etc.

Cluster Logging attempts to parse and normalize the log data, but there are some cases where it cannot, especially if you use the `MERGE_JSON_LOG` feature.
```
[1],[2] are footnotes or some sort of links to https://github.com/openshift/origin-aggregated-logging/blob/master/docs/com.redhat.viaq-openshift-operations.asciidoc and https://github.com/openshift/origin-aggregated-logging/blob/master/docs/com.redhat.viaq-openshift-project.asciidoc

Or perhaps we already document the fields available for searching and listing in Kibana in our docs?  If so, then just link that here.

Reply:
Thanks!  1a00924166992d44b90ddeb28f3f887a1fb351e9